tortoise.py

```diff
@@ -1,8 +1,18 @@
+import sys
+
+
 def main():
-    FILENAME = 'tiny.tortoise'
+    
+    try:
+        FILENAME = sys.argv[1]
+    except IndexError:
+        print('\nYou have to specify what tortoise you want executed.\
+              \nPlease add it as an argument on the command line.\
+              \nYour choices are: tiny.tortoise, medium.tortoise, or large.tortoise.\n')
+        return
 
-    with open(FILENAME, 'r') as tiny_tortoise:
-        tortoise_program = tiny_tortoise.readlines()
+    with open(FILENAME, 'r') as tortoise:
```

Comment:
I was going to comment on the fact this used to be called `tiny_tortoise`. Good job on spotting and changing it :+1: